packages/block-library/src/button/style.scss

```diff
@@ -34,7 +34,6 @@ $blocks-block__margin: 0.5em;
 	}
 }
 
-// Increased specificity needed to override margins
```

Comment:
I guess the margins are gone, but is the specificity still needed for the gap?

Reply:
I think we can remove this, yep. Thanks!